### PR TITLE
Phone number detector bulk messages

### PR DESCRIPTION
**Pull request by prik2693**
API to detect phone number entity for bulk input messages.

### CODE REVIEW

**Comment by srajdev** [comment]
:+1: No lint errors found.

File: ner_v2/api.py [hunk 1]
```diff
@@ -397,6 +397,7 @@ def number_range(request):
     return HttpResponse(json.dumps({'data': entity_output}), content_type='application/json')
 
 
+@csrf_exempt
```
Comment (amansrivastava17) [inline ner_v2/api.py]: Update docs, Message can be list or str.  Also response will be list of list for bulk call.

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

File: ner_v2/api.py [hunk 2]
```diff
@@ -397,11 +397,12 @@ def number_range(request):
...
 def phone_number(request):
     """Uses PhoneDetector to detect phone numbers
 
         request params:
-            message (str): natural text on which detection logic is to be run. Note if structured value is
```
Comment (amansrivastava17) [inline ner_v2/api.py]: can be list or str, depending on requests

Comment (amansrivastava17) [reply ner_v2/api.py]: Also add Examples for  post request

```message = ["Call 02226129857' , 'message +1(408) 92-124' ,'send 100rs to 91 9820334416 9920441344']
entity_name = 'phone_number'
source_language = 'en'

entity_output:
[
   [{
        "detection": "message",
        "original_text": "91 9820334416",
        "entity_value": {
            "value": "919820334416"
        },
        "language": "en"
        }
    ],
    [
        {
            "detection": "message",
            "original_text": "9920441344",
            "entity_value": {
                "value": "9920441344"
            },
            "language": "en"
        }
    ],
    [
        {
            "detection": "message",
            "original_text": "02226129857",
            "entity_value": {
                "value": "02226129857"
            },
            "language": "en"
        },
        {
            "detection": "message",
            "original_text": "+1(408) 92-124",
            "entity_value": {
                "value": "140892124"
            },
            "language": "en"
        }
    ]
]
```

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Pull request merged**